Dockerfile.apidocs

```diff
@@ -0,0 +1,51 @@
+# Sentry api documentation generation environment
+#
+# Instructions:
+#
+#   Build the container:
+#     $ docker build -t sentry:apidocs -f Dockerfile.apidocs .
+#   Run the container:
+#     $ docker run --rm -v $(pwd):/usr/src/output sentry:apidocs
+#
+# The container will dump the generated documentation in markdown and JSON
+# formats into the /usr/src/output directory which you should mount as a volume
+#
+FROM python:2.7.11-slim
+
+RUN groupadd -r redis --gid=998 && useradd -r -g redis --uid=998 redis \
```

Comment:
None of this matters here. You're also creating a postgres user and we don't install Postgres.